src/plugin.js

```diff
@@ -132,18 +84,36 @@ module.exports = ({ path, packagePath, urls, imports } = {}) => {
                 // Cache we've processed this
                 processed.set(decl, true);
             };
+
+            const mapping = new Map();
+
             return {
                 // Run initially once, this is to ensure it runs before postcss-import
                 async Once(root) {
-                    const mapping = await mapFetch;
+                    // Load eik config from eik.json or package.json
+                    const config = await helpers.getDefaults(path);
+                    const cUrls = [];
+                    for (const map of config.map) {
+                        cUrls.push(map);
+                    }
```

Comment:
Looks like cUrls is only being used below on with `...cUrls` this should copy the array so I think you can skip the for loop.